File: Parrot.java
import java.util.ArrayList;

public class Parrot {
    private ArrayList<String> phrases = new ArrayList<>();
    // private String phrase;

    public Parrot(String phrase) {
        // this.phrase = phrase;
        this.phrases.add(phrase);
    }
    
    public Parrot(ArrayList<String> phrases) {
        this.phrases.addAll(phrases);
    }

    public void say(int howManyTimes) {
        int index = (int)(Math.random() * this.phrases.size());
        for(int i = 0; i < howManyTimes; i++) {
            // System.out.println(phrase);
            System.out.println(phrases.get(index));
            
        }
    }

    /* public void setPhrase(String new_phrase) {
        this.phrase = new_phrase;
    } */

    public void addPhrase(String phrase) {
        this.phrases.add(phrase);
    }

    public void addPhrases(ArrayList<String> phrases) {
        this.phrases.addAll(phrases);
    }

    public static void main(String[] args) {
        Parrot p = new Parrot("Wow, green");
        p.addPhrase("Bruh");
        p.addPhrase("Not");
        p.addPhrase("Bread");

        for(int i = 0; i < 8; i++) {
            p.say(1);
        }
    }
}

File: parrot.cpp
#include <iostream>
#include <vector>

using std::string, std::vector, std::cout, std::endl;

class Parrot {
    private:
        vector<string> phrases;
        // string phrase;
    public:
        Parrot(const string phrase) {
            this->phrases.push_back(phrase);
            // this->phrase = phrase;
        }

        void say(const unsigned int howManyTimes) {
            int index = rand() % this->phrases.size();
            for(unsigned int i = 0; i < howManyTimes; i++) {
                cout << phrases[index] << "\n";
            }
        }

        /* void setPhrase(string new_phrase) {
            this->phrase = new_phrase;
        } */

        inline void addPhrase(const string phrase) {
            this->phrases.push_back(phrase);
        }

        inline void addPhrases(const vector<string> phrases) {
            for(string phrase : phrases) {
                this->phrases.push_back(phrase);
            }
        }
};

int main() {
    Parrot p("Wow, green");
    p.addPhrase("Bruh");
    p.addPhrase("Not");
    p.addPhrase("Bread");

    for(int i = 0; i < 8; i++) {
        p.say(1);
    }

    return 0;
}